addons/google_calendar/models/calendar_recurrence_rule.py

```diff
@@ -107,10 +107,10 @@ def _write_from_google(self, gevent, vals):
                     partner.name = attendee[2].get('displayName')
 
         for odoo_attendee_email in set(existing_attendees.mapped('email')):
-            # Remove old attendees
+            # Remove old attendees. Sometimes, several partners have the same email.
             if email_normalize(odoo_attendee_email) not in emails:
-                attendee = existing_attendees.filtered(lambda att: att.email == email_normalize(odoo_attendee_email))
-                self.calendar_event_ids.write({'need_sync': False, 'partner_ids': [(3, attendee.partner_id.id)]})
+                attendee = existing_attendees.exists().filtered(lambda att: att.email == email_normalize(odoo_attendee_email))
+                self.calendar_event_ids.write({'need_sync': False, 'partner_ids': [(3, att.partner_id.id) for att in attendee]})
```

Comment:
Even though `fields.Command`'s syntax is not very compact, it helps using explicitly named constants instead of magic numbers.